src/css/index.scss

```diff
@@ -0,0 +1,230 @@
+* {
+  box-sizing: border-box;
+  font-family: 'montserrat', sans-serif;
```

Comment:
Is `montserrat` a normal web font? Just want to know the reason why you prioritize this font :)

Reply:
All the CSS I've written is using Montserrat and I am ok with the way it looks like.

Reply:
That's cool. But please consider installing [Montserrat](https://fonts.google.com/specimen/Montserrat) in the HTML head via Google Front, or other users may not install that font in their system, so the font will fall back to the default `sans-serif`.

```HTML
<html>
  <head>
    <link href="https://fonts.googleapis.com/css?family=Montserrat:400,700&display=swap" rel="stylesheet">
  </head>
  <body>
  </body>
</html>
```